package.json

```diff
@@ -22,7 +22,7 @@
   "productName": "Hathor Wallet",
   "description": "Light wallet for Hathor Network",
   "author": "Hathor Labs <contact@hathor.network> (https://hathor.network/)",
-  "version": "0.27.1-rc1",
+  "version": "0.27.1-rc2",
```

Comment:
Should the release-candidate versions be using the following syntax `rc.N`?

This syntax is compliant with the semver. Also, it is describe in the [release guide for hathor-wallet](https://github.com/HathorNetwork/ops-tools/blob/f6f3bba8cfb89020a6b169fb52bb9fa43127ad23/docs/release-guides/hathor-wallet.md#release-candidate).